docs/developer/extending/apps/asynchronous-webhooks.mdx

```diff
@@ -56,6 +56,8 @@ Deprecated headers list:
 
 :::
 
+Saleor gives you an option to add custom headers to the request (available since version 3.12). Currently supports `Authorization*` and `X-*` keys with limitation of 5 headers per webhook and 998 characters per header. To check how to add `customHeaders` go to [Subscribing to webhook](/docs/developer/extending/apps/asynchronous-webhooks.mdx#subscribing-to-a-webhook).
+
```

Comment:
```suggestion
Saleor gives you an option to add custom headers to the request (available since version 3.12). Currently supports `Authorization*` and `X-*` keys with limitations of 5 headers per webhook and 998 characters per header. To check how to add `customHeaders` go to [Subscribing to webhook](/docs/developer/extending/apps/asynchronous-webhooks.mdx#subscribing-to-a-webhook).

```